makerpms.sh

```diff
@@ -4,7 +4,26 @@ set -o errexit
 pushd "$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 
 test ! -x "configure" && autoreconf -i
-test ! -f "Makefile" && ./configure "$@"
+# run configure with the same parameters as RPM build
+# this makes it easy to tweak files locally and use make install
+test ! -f "Makefile" && ./configure --enable-silent-rules \
+	--host=$(rpm -E %{_host}) \
```

Comment:
@tiran can you explain how a script with the name **makerpms.sh** cannot be rpm specific?